pkg/operator/starter.go

```diff
@@ -41,89 +34,66 @@ const (
 )
 
 func RunOperator(ctx context.Context, controllerConfig *controllercmd.ControllerContext) error {
-	// Kubernetes client, used to manipulate StorageClasses
-	kubeClient, err := kubernetes.NewForConfig(controllerConfig.ProtoKubeConfig)
-	if err != nil {
-		return err
-	}
-	kubeInformers := informers.NewSharedInformerFactory(kubeClient, resync)
-
-	// operator.openshift.io client, used to manipulate the operator CR
-	operatorClientSet, err := opclient.NewForConfig(controllerConfig.KubeConfig)
-	if err != nil {
-		return err
-	}
-	operatorInformers := opinformers.NewSharedInformerFactoryWithOptions(operatorClientSet, resync,
-		opinformers.WithTweakListOptions(singleNameListOptions(operatorclient.GlobalConfigName)),
```

Comment:
Is singleNameListOptions still used?

Reply:
It is, in `csoclients.go`. And it is wrong, now it must watch also CSI driver CRs which have various names. Fixed to watch everything :-(

Reply:
I don't follow, I grepped `singleNameListOptions` in your branch and only found 2 definitions of this method, but they aren't used anywhere. Am I missing anything?

```go
$ ag singleNameListOptions
pkg/operator/starter.go
112:func singleNameListOptions(name string) func(opts *metav1.ListOptions) {

pkg/csoclients/csoclients.go
107:func singleNameListOptions(name string) func(opts *metav1.ListOptions) {
```

Reply:
I removed it because now the operator must watch all ClusterCSIDrivers, regardless of their name.